examples/app-claim.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: platform-ref-aws-ghost
   namespace: default
   annotations:
-    uptest.upbound.io/pre-delete-hook: testhooks/delete-release-app.sh
+    uptest.upbound.io/pre-delete-hook: testhooks/delete-release.sh
```

Comment:
why do we need to change the test hook?

Reply:
answer is in this comment: https://github.com/upbound/platform-ref-aws/pull/110#discussion_r1266910533